Makefile

```diff
@@ -37,6 +37,24 @@ submit: plugin
 	@curl -H "Authorization: Bearer ${DKU_PLUGIN_DEVELOPER_TOKEN}>" -X PUT ${artifact_repo_target} -T dist/${archive_file_name}
 	@echo "[SUCCESS] Publishing archive to artifact repository: Done!"
 
+unit-tests:
+	@echo "[START] Running unit tests..."
+	@( \
+		python3 -m venv env/; \
+		source env/bin/activate; \
```

Comment:
Following Nico recommandations, replace lines from 45 to 47 by:
```
		pip3 install --upgrade pip;\
		pip install --no-cache-dir -r tests/python/requirements.txt; \
		pip install --no-cache-dir -r code-env/python/spec/requirements.txt; \
		export PYTHONPATH="$(PYTHONPATH):$(PWD)/python-lib"; \
                pytest --junitxml=unit.xml tests/python/unit || true; \
```
The junit options ensure that the tests' results are in the surefire format, the format Jenkins can understand